internal/config/config.go

```diff
@@ -412,8 +412,35 @@ func (c *Config) LoadDirectory(path string) error {
 	return nil
 }
 
+// Try to find a default config file at these locations (in order):
+//   1. $TELEGRAF_CONFIG_PATH
+//   2. $HOME/.telegraf/telegraf.conf
+//   3. /etc/telegraf/telegraf.conf
+//
+func getDefaultConfig() (string, error) {
```

Comment:
`getDefaultConfig` -> `getDefaultConfigPath`

The former implies this loads the config rather than the path while the latter is about the path location.